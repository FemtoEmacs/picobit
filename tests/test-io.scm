;; test IO

(putchar (getchar))

(getchar-wait 10)

(sleep 10)

