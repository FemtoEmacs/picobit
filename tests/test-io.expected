:0080000080
:10800000FBD7000102B4003C50B5A14061EE0EB5B3
:1080100096EE0EB582EED801E121C1B68001E121D4
:10802000C89320B67800EF012007E5EF000407E5CC
:1080300002B400357101209220EFB670012007E6EE
:0280400000EF4F
:00000001FF
